rasa/core/utils.py

```diff
@@ -30,24 +30,32 @@
 
 
 def configure_file_logging(
-    logger_obj: logging.Logger, log_file: Optional[Text]
+    logger_obj: logging.Logger, log_file: Optional[Text], use_syslog: Optional[bool]
 ) -> None:
     """Configure logging to a file.
 
     Args:
         logger_obj: Logger object to configure.
         log_file: Path of log file to write to.
+        use_syslog: Add syslog as a logger
     """
-    if not log_file:
+    if not log_file and not use_syslog:
```

Comment:
Do we need this if both the subsequent statements are `if`s?

Reply:
You are right. I fixed that

Reply:
I did this way to be as close as possible from the original file. But if you say that the None return has no impact, I can fix that